example.py

```diff
@@ -1,18 +1,30 @@
+import keyring;
```

Comment:
Interesting.... Not sure if this makes sense to be added to the framework code since this is more of a application level implementation. Thoughts?